nion/swift/model/DocumentModel.py

```diff
@@ -2363,8 +2384,14 @@ def __make_computation(self, processing_id: str, inputs: typing.List[typing.Tupl
                     # TODO: how to refer to the data_items? hardcode to data_item0 for now.
                     display_item0 = self.get_display_item_for_data_item(data_item0)
                     display_data_channel0 = display_item0.display_data_channel if display_item0 else None
-                    connection = Connection.PropertyConnection(display_data_channel0, connection_src_prop, new_regions[connection_dst], connection_dst_prop, parent=new_data_item)
-                    self.append_connection(connection, project=project)
+                    connection = None
+                    if "region" in connection_dst:
+                        connection = Connection.PropertyConnection(display_data_channel0, connection_src_prop, new_regions[connection_dst], connection_dst_prop, parent=new_data_item)
+                    elif connection_dst == "display_data_channel":
+                        dst_display_data_channel = new_display_item.display_data_channel
+                        connection = Connection.PropertyConnection(display_data_channel0, connection_src_prop, dst_display_data_channel, connection_dst_prop, parent=new_data_item)
+                    if connection:
+                        self.append_connection(connection, project=project)
```

Comment:
This code is confusing -- and I think it fails for the `map_connection` which has a `connection_dst` of 'pick_point'. Can you put the `connection_dst == "display_data_channel"` first and then use `else` for the original version? This might need some tests.

Reply:
I made the change you requested. As far as I can see the tests should be written for `nion/swift/model/Processing.py`, right? At least if we want to test that `map_connections` are established correctly. I couldn't find any place where the `Processing.py` code is actually used, so I'm not sure how exactly to write tests for that.

Reply:
Processing is used for the FFT windows at the moment; but they don't hit this code. I have an internal branch that uses this code - so I will add tests for it. Thanks.